assets/src/design-system/theme/colors.js

```diff
@@ -98,6 +98,10 @@ const standard = {
   black: '#000',
   white: '#FFF',
 };
+const disable = {
+  dark: '#2F3131',
+  light: '#E4E4E1',
+};
```

Comment:
i know this is from a slack convo between you and @ivayloklissarov  but i am kind of wary of introducing more colors aside from the brand palette? 

looking at the design system, it _looks like_ the dark interactive background disable gray 80 has been updated to `#2F3131`  while the actual gray 80 shade is still #26292A  😅  

from what @samitron7 's said in the past about the brand colors being the gold standard when there's a discrepancy, i think that we should leave the `interactiveBg.disable` as it is. If we need to change the disabled color we should do it from the selection of brand grays we have and if a new gray needs to be added it should be done separate. 

Feels like a small thing, won't block it, but I think perhaps there's some coloring here that is, pardon the pun, a gray area. 
<img width="591" alt="Screen Shot 2021-03-19 at 8 56 22 AM" src="https://user-images.githubusercontent.com/10720454/111808827-98324980-8891-11eb-81db-0dbb69a0e3f5.png">
<img width="485" alt="Screen Shot 2021-03-19 at 8 58 14 AM" src="https://user-images.githubusercontent.com/10720454/111808832-98cae000-8891-11eb-8ce6-b8ed7049e63c.png">
<img width="408" alt="Screen Shot 2021-03-19 at 8 56 59 AM" src="https://user-images.githubusercontent.com/10720454/111808834-98cae000-8891-11eb-93f6-0d3de5a9ff41.png">

Reply:
> looking at the design system, it _looks like_ the dark interactive background disable gray 80 has been updated to `#2F3131` while the actual gray 80 shade is still #26292A 😅
> 
> from what @samitron7 's said in the past about the brand colors being the gold standard when there's a discrepancy, i think that we should leave the `interactiveBg.disable` as it is. If we need to change the disabled color we should do it from the selection of brand grays we have and if a new gray needs to be added it should be done separate.

Yeah, I'll revert this (especially since the disabled colors aren't shown anywhere) and we can chat about this more in UAT